src/main/java/com/tradeshift/resources/AppConfig.java

```diff
@@ -0,0 +1,52 @@
+package com.tradeshift.resources;
+
+import com.tradeshift.service.HelloWorldService;
+import com.tradeshift.service.HelloWorldServiceImpl;
+import com.tradeshift.service.MessagesDAO;
+import com.tradeshift.service.MessagesDAOImpl;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.datasource.SimpleDriverDataSource;
+
+import javax.sql.DataSource;
+
+@Configuration
+public class AppConfig{
+
+    @Bean(name = "helloWorldService")
+    public HelloWorldService getHelloWorldService() {
+
+        return new HelloWorldServiceImpl();
+    }
+
+    @Bean
+    DataSource getDataSource() {
+        String addr = System.getenv("DB_PORT_5432_TCP_ADDR");
+        String port = System.getenv("DB_PORT_5432_TCP_PORT");
+        if (addr == null || port == null) {
+            throw new IllegalArgumentException("Can not connect to postgress using variables  "  + "\n env:" + System.getenv());
+        }
+
+        SimpleDriverDataSource dataSource = new SimpleDriverDataSource(new org.postgresql.Driver(), "jdbc:postgresql://"+ addr+":"+port+ "/messagesdb", "messagesdb", "password");
+        String sql = "DROP TABLE IF EXISTS message;\n" +
```

Comment:
Sure I Can do the other way around to create the table if it doesn't exist. But there was no specification about it and it's a neat way to test a fresh db :P I'll change it then